packages/graphql/src/schema-model/parser/annotations-parser/limit-annotation.ts

```diff
@@ -19,27 +19,26 @@
 
 import type { DirectiveNode } from "graphql";
 import { Neo4jGraphQLSchemaValidationError } from "../../../classes";
-import { QueryOptionsAnnotation } from "../../annotation/QueryOptionsAnnotation";
+import { LimitAnnotation } from "../../annotation/LimitAnnotation";
 import { parseArguments } from "../parse-arguments";
-import { queryOptionsDirective } from "../../../graphql/directives";
+import { limitDirective } from "../../../graphql/directives";
 
-export function parseQueryOptionsAnnotation(directive: DirectiveNode): QueryOptionsAnnotation {
-    const { limit } = parseArguments(queryOptionsDirective, directive) as {
-        limit: {
-            default?: number;
-            max?: number;
-        };
+export function parseLimitAnnotation(directive: DirectiveNode): LimitAnnotation {
+    const { default: _default, max } = parseArguments(limitDirective, directive) as {
```

Comment:
**nitpick** why are we renaming this one to `_default`?

Reply:
Because `default` is a reserved word in JavaScript 🤦